coord/src/dist_aggregate.c

```diff
@@ -75,14 +75,26 @@ static bool getCursorCommand(MRReply *res, MRCommand *cmd, MRIteratorCtx *ctx) {
 static int netCursorCallback(MRIteratorCallbackCtx *ctx, MRReply *rep) {
   MRCommand *cmd = MRIteratorCallback_GetCommand(ctx);
 
+  // If the root command of this reply is a DEL command, we don't want to
+  // propagate it up the chain to the client
+  if (cmd->rootCommand == C_DEL) {
+    if (MRReply_Type(rep) == MR_REPLY_ERROR) {
+      RedisModule_Log(NULL, "warning", "Error returned for CURSOR.DEL command from shard");
+    }
+    // Discard the response, and return REDIS_OK
+    MRReply_Free(rep);
+    MRIteratorCallback_Done(ctx, 1);
```

Comment:
nitpicking, but `1` here indicates an error, although nobody uses this value inside the function. Consider this change:
```suggestion
    MRIteratorCallback_Done(ctx, MRReply_Type(rep) == MR_REPLY_ERROR);
```